### PR TITLE
changed the select2 color to match the theme(fixes issue #63)

### PR DESCRIPTION
**Pull request by ramyaparimi**
This PR is created to fix the color of select2 box to match mote blue theme.

This is created for [issue#63](https://github.com/fedora-infra/mote/issues/63).

**Current select2 color**
image

**Proposed select2 color to match the theme:**
image

### CODE REVIEW

**Review by nasirhm — APPROVED** [review]
LGTM Thank you for working on it @ramyaparimi, 

Here's the diff:
Screenshot from 2020-11-12 12-45-39

**Review by t0xic0der — APPROVED** [review]
+1x LGTM from my end too. I didn't see the code differences though but with a change as small as this, I think a screenshot review should suffice.

**Review by t0xic0der — CHANGES_REQUESTED** [review]
Umm... Wait.

Is it possible to introduce changes via inline CSS @ramyaparimi by overriding the classes specified in the `select2.min.css`? I would not want to make changes in the importable CSS file and overriding the classes for making contained changes is a lot more acceptable way of doing things here.

**Comment by ramyaparimi** [comment]
> Umm... Wait.
> 
> Is it possible to introduce changes via inline CSS @ramyaparimi by overriding the classes specified in the `select2.min.css`? I would not want to make changes in the importable CSS file and overriding the classes for making contained changes is a lot more acceptable way of doing things here.

+1
I get what you mean! I made inline changes now. Let me know what you think.

**Comment by ramyaparimi** [comment]
Thank you @nasirhm @t0xic0der for reviewing the PR.
I made added the color in mote.css that would reflect inline. Let me know what you think about it 😃

File: mote/static/css/mote.css
Comment (t0xic0der) [inline mote/static/css/mote.css:282]: Yes, that's exactly what I was conveying. Making changes to this custom-made CSS file would be much better than adding those changes to an imported one for who knows Select2 might be used somewhere else and not need those theming options.

If the consequence of these changes look the same as before, I think it is ready for a merge.

**Review by t0xic0der — COMMENTED** [review]
Let me quickly hitch it up locally and approve it once I am done checking.

**Review by t0xic0der — APPROVED** [review]
LGTM.

File: mote/static/css/mote.css
Comment (ramyaparimi) [reply mote/static/css/mote.css:282]: This is giving out the same output!! Thanks for reviewing!

**Comment by nasirhm** [comment]
Thank You for working on it @ramyaparimi, It looks awesome, Can you please squash these changes into one to ease the merge ?

**Comment by ramyaparimi** [comment]
umm... so I used the squash option here and accidentally squashed and merged 🤦

**Pull request merged**
